src/Service/Mailer/Mail/AbstractNotificationMailer.php

```diff
@@ -99,6 +99,12 @@ public function generateLink(string $route, array $params): string
         return $this->parameterBag->get('host_url').$this->urlGenerator->generate($route, $params);
     }
 
+    public function generateAbsoluteLink(string $route, array $params): string
+    {
+        return $this->parameterBag->get('host_url').$this->urlGenerator->generate($route, $params,
```

Comment:
Sans le $this->parameterBag->get('host_url')

Reply:
ok c'est traité